poc/policy/policy.py

```diff
@@ -0,0 +1,48 @@
+"""
+Interface for DP policies.
+
+A DP policy relies on all entities of an aggregation protocol to add the right
```

Comment:
Actually this depends on the threat model right? E.g., for HistogramWithAggregatorDp we need only one Aggregator to be honest, but we lose some utility if both Aggregators are honest.